### ISSUE TITLE
Remove .classpath and .project files from git

### ISSUE THREAD

**Issue by ThadHouse** [ORF-4450/Robot22B#1]
These files are generated by the VS Code Java extension, and should not be pushed to source control. We've seen these being in source control cause a lot of weird issues.

**Comment by Crossle86** [comment]
Thad, will do. Another question: I am on teamforge looking at the simulation start up fail issue I posted there. There are several replies and I want to respond...but today when I log on and look at that discussion, there are  no buttons to post a  reply or new comment. This worked fine when I posted on Friday but not today.

**Comment by Crossle86** [comment]
Never mind...having an idiot moment.

**Comment by ThadHouse** [comment]
Yeah we noticed that too. For some reason that thread is being weird. We can't reply either.

You can create a new discussion with the responses that's fine.

**Comment by Crossle86** [comment]
The reason is that one of the previous posts is very wide and now pushed the buttons off the screen for all posts. I failed to notice the scroll bar until just now.

**Comment by ThadHouse** [comment]
Oh. That makes sense.
